parsing/lexer.mll

```diff
@@ -684,7 +703,7 @@ and comment = parse
       }
   | newline
       { update_loc lexbuf None 1 false 0;
-        store_lexeme lexbuf;
+        store_normalized_newline ();
         comment lexbuf
```

Comment:
Note: newlines are also normalized in comments, because (some) comments are now stored as documentation attributes in the AST, so the same portability issue as string literals could arise.

Reply:
Interesting point. That of course affects ocamldoc's verbatim and raw backends sections, but is again likely more an improvement than a problem.